code/lib/builder-vite/src/plugins/mdx-plugin.ts

```diff
@@ -15,10 +15,8 @@ const isStorybookMdx = (id: string) => id.endsWith('stories.mdx') || id.endsWith
 export async function mdxPlugin(options: Options): Promise<Plugin> {
   const include = /\.mdx?$/;
   const filter = createFilter(include);
-  const addons = await options.presets.apply<StorybookConfig['addons']>('addons', []);
```

Comment:
@shilman addons wasn't defined when doing this. When I debugged the code it seems like I can get the options directly?